modules/compiler/targets/host/source/info.cpp

```diff
@@ -130,11 +130,11 @@ void HostInfo::get(compiler::AddCompilerFn add_compiler) {
       HOST_CROSS_DEVICE_NAME_AARCH64));
 #endif
 #ifdef HOST_CROSS_X86
-  add_compiler(getCrossCompilerInfo<host::arch::X86, HOST_OS>(
+  add_compiler(getCrossCompilerInfo<host::arch::X86, host::os::LINUX>(
```

Comment:
The title is a bit misleading given this change as well.

Reply:
The goal here is to disable a test but agreed it doesn't exactly disable a test. Without this, we get

```console
% bin/UnitCompiler
[...]
[----------] 3 tests from CreateBinaryTest
[ RUN      ] CreateBinaryTest.CreateBinary/ComputeAorta_AArch64
[       OK ] CreateBinaryTest.CreateBinary/ComputeAorta_AArch64 (24 ms)
[ RUN      ] CreateBinaryTest.CreateBinary/ComputeAorta_x86_64
Assertion failed: (host_device_info.native && "macOS cross-compile not supported"), function initWithBuiltins, file target.cpp, line 244.
zsh: abort      bin/UnitCompiler
```

I'll try to reword.

Reply:
Couldn't come up with wording that covered both, so just edited the title to separate the two changes.